openmetadata-ui/src/main/resources/ui/src/axiosAPIs/ingestionPipelineAPI.ts

```diff
@@ -88,3 +88,9 @@ export const getIngestionPipelineLogById = (
 ): Promise<AxiosResponse> => {
   return APIClient.get(`/services/ingestionPipelines/logs/${id}/last`);
 };
+
+export const killIngestionPipelineById = (
```

Comment:
Let's start adding: `get`, `post`, `put`, `delete` for Rest API functions, so that it will differ from normal function